test/unittests/t_catalog_mgr.cc

```diff
@@ -20,7 +20,6 @@ using namespace std;  // NOLINT
 namespace catalog {
 
 class T_CatalogManager : public ::testing::Test {
-
```

Comment:
Never mind, I'm creating Unit tests for the balancer, so this file can be included